scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -32,7 +38,7 @@ def generate_tmcf(self):
                     'observationDate: C:pcm-dpc->Date\n'
                     'value: C:pcm-dpc->{SVname}\n\n')
         idx = 1
-        with open(self.name + '.tmcf', 'a') as f_out:
+        with open(out, 'a') as f_out:
```

Comment:
curious, why not just replace 'a' with 'w' ?

Reply:
`geo_template` first writes to the output file, and then `generate_tmcf` appends to it

Reply:
Perhaps geo_template could be passed the file handle to which to write.